integrations/extensions/starter-kits/testitall/mock-server/server/controllers/delay-controller.js

```diff
@@ -0,0 +1,18 @@
+export function delay(req, res) {
+  let sleep_duration = req.body.sleep_time || 3;
+
+  sleep(sleep_duration * 1000);
```

Comment:
I know this is just demo code, but its best practice in JS to not try to fill up the call stack like this to force a sleep, but to use async code. Something along the lines of...
```
export async function delay(req, res) {
  let sleep_duration = req.body.sleep_time || 3;
 await sleep(sleep_duration * 1000)
return res.json({ status: 'UP', sleep_duration });
}

async function sleep(milliseconds) {
     const promise = new Promise();
     setTimeout(() => {
          promise.resolve();
     }, milliseconds);
     return promise;
}
```

Reply:
Thanks for your suggestion! Just updated the code with your change, and need someone with write access to perform merge.